Directory.Packages.props

```diff
@@ -0,0 +1,103 @@
+﻿<Project>
+  <ItemGroup>
+    <PackageVersion Include="BenchmarkDotNet" Version="$(BenchmarkDotNetPackageVersion)" />
```

Comment:
Do we need to still use all of these version variables, if all of the versions are neatly specified in this file? Can we not get rid of (most of?) Versions.props now?

Reply:
Think we can get rid of everything in Version.props that is manually updated. There are still some variables that come from automation that need to be kept.

Reply:
Okay that exercise revealed a lot of unused properties and some odd decisions around versions. Def worth the time to have done that.